oelint_adv/core.py

```diff
@@ -230,8 +228,8 @@ def create_lib_arguments(files: List[str],
     parser.add_argument('--quiet', action='store_true', default=False)
     parser.add_argument('--jobs', type=int, default=mp.cpu_count())
     parser.add_argument('--color', action='store_true', default=False)
-    parser.add_argument('--noinfo', action='store_true', default=False)
-    parser.add_argument('--nowarn', action='store_true', default=False)
+    parser.add_argument('--hide', action='append', default=None,
+                        choices=['neutral', 'info', 'warning', 'error'])
```

Comment:
What is `neutral`?

Reply:
ahh I get the intention, but no, I really want to stick to the 3 categories "error", "warning", "info".
With the combination of the new --hide parameter and a rule file, I don't see any case not covered, so a neutral category is not needed IMO.
From my perspective "neutral" equal "info" as the least interesting findings of a run

Reply:
> What is `neutral`?

https://github.com/priv-kweihmann/oelint-adv/blob/0c463d615d02d68fb025ee353f9b7ab644e21cda/oelint_adv/state.py#L33-L39

To be fair this was the only existing reference I found to severity neutral and funny enough, given your other comments I didn't want to break compatibility :) but looking at it closer I suppose the reference to neutral is an error in this case?

Reply:
Ahh the color codes ... In that case it makes sense to have a fallback for miscoded custom rules - but still not a valid severity 😁